spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/RibbonCommandContext.java

```diff
@@ -43,30 +44,33 @@
 	private final List<RibbonRequestCustomizer> requestCustomizers;
 	private InputStream requestEntity;
 	private Long contentLength;
+	private Object loadBalancerKey;
 
 	/**
+
 	 * Kept for backwards compatibility with Spring Cloud Sleuth 1.x versions
 	 */
 	@Deprecated
 	public RibbonCommandContext(String serviceId, String method,
 								String uri, Boolean retryable, MultiValueMap<String, String> headers,
 								MultiValueMap<String, String> params, InputStream requestEntity) {
 		this(serviceId, method, uri, retryable, headers, params, requestEntity,
-			new ArrayList<RibbonRequestCustomizer>(), null);
+			new ArrayList<RibbonRequestCustomizer>(), null, null);
 	}
 
 	public RibbonCommandContext(String serviceId, String method, String uri,
 								Boolean retryable, MultiValueMap<String, String> headers,
 								MultiValueMap<String, String> params, InputStream requestEntity,
 								List<RibbonRequestCustomizer> requestCustomizers) {
 		this(serviceId, method, uri, retryable, headers, params, requestEntity,
-			requestCustomizers, null);
+			requestCustomizers, null, null);
 	}
 
 	public RibbonCommandContext(String serviceId, String method, String uri,
```

Comment:
To avoid breaking anyone I think we should create a new constructor with the new parameter.  You could also mark the old constructor deprecated if it is not necessary anymore.

Reply:
Okay. I will add a new constructor and some documentation soon.